src/main/java/org/candlepin/pki/impl/JSSProviderLoader.java

```diff
@@ -109,7 +61,7 @@ public static String getJSSVersion() {
      * Initializes the JSS CryptoManager (which in turn initializes the NSS DB), and loads the JSS provider
      * into the JVM. The method should be called during context initialization. Can be called more than once
      * without ill effect (-1 will be returned if the provider is already installed).
-     *
+     * <p>
      * Note: This method uses reflection to initialize the CryptoManager, due to dynamically loading the
      * InitializationValues class, depending on which JSS version is on the classpath. This is because of
      * breaking changes introduced between JSS 4.4.X (RHEL 7) and 4.5.0+ (RHEL 8).
```

Comment:
Lets remove this now since it is irrelevant

Reply:
Done.